### PR TITLE
[master] Webhook payload samples

### PR DESCRIPTION
**Pull request by icehess**

### CODE REVIEW

File: applications/crossbar/src/modules/cb_webhooks.erl [hunk 1]
```diff
@@ -356,6 +394,51 @@ summary_available(Context) ->
...
+        {'error', 'not_found'} ->
+            crossbar_util:response_bad_identifier(WebhookName, Context);
+        {'error', _} ->
+            cb_context:add_system_error('datastore_fault', Context)
+    end.
+
+-spec fetch_webhook_samples(kz_term:ne_binary() | kz_term:ne_binaries()) ->
+                                   {'ok', kz_json:objects()} |
+                                   {'error', any()}.
+fetch_webhook_samples(<<"webhooks_all">>) ->
```
Comment (jamesaimonetti) [inline applications/crossbar/src/modules/cb_webhooks.erl]: I think having to fetch `/v2/.../webhooks/samples/all` is a bit strange vs listing available samples via `/v2/.../webhooks/samples/` then querying the specific sample you want to see via `/webhooks/samples/{SAMPLE_ID}`.

**Comment by lazedo** [comment]
@icehess can you address @jamesaimonetti comment and rebase ? thanks

File: applications/crossbar/src/modules/cb_webhooks.erl [hunk 2]
```diff
@@ -193,6 +237,8 @@ validate_webhooks(Context, ?HTTP_PATCH) ->
 -spec validate(cb_context:context(), path_token()) -> cb_context:context().
 validate(Context, ?PATH_TOKEN_ATTEMPTS) ->
     summary_attempts(Context, 'undefined');
+validate(Context, ?PATH_TOKEN_SAMPLES) ->
+    fetch_webhook_samples(Context, <<"webhooks_all">>);
```
Comment (lazedo) [inline applications/crossbar/src/modules/cb_webhooks.erl]: shouldn't this return a list of available webhooks instead of `all` webhooks ?

Comment (icehess) [reply applications/crossbar/src/modules/cb_webhooks.erl]: GET `/webhooks` is returning all available webhooks. This one is for fetching all samples in request per what aux team (or ui tema? idk) requested

Comment (lazedo) [reply applications/crossbar/src/modules/cb_webhooks.erl]: it should be the list of samples not all the samples

**Pull request merged**